### PR TITLE
Detail 1.0:Built the detail page according to Figma

### PR DESCRIPTION
**Pull request by YiyuTimTian**
I have made the new detail page, which is exactly like the design on Figma. It works well on my laptop. Feel free to merge it with our main branch.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -77,13 +78,14 @@ function App() {
   const debouncedSearchTermChanged = debounce(search);
 
   return (
-    <div className="App bg-gray-100">
-      <Header />
+    <div className="App">
```
Comment (steve8circle) [inline src/App.js]: so we will have header/footer in the list page, not on the detail page?

File: src/detail/detail.css [hunk 2]
```diff
@@ -0,0 +1,256 @@
+.Layout{
```
Comment (steve8circle) [inline src/detail/detail.css:1]: can we use more tailwind instead?

File: src/detail/more/more.css [hunk 3]
```diff
@@ -0,0 +1,94 @@
+.caro{
```
Comment (steve8circle) [inline src/detail/more/more.css]: same as last comment. let's try more with tailwind.

**Review by steve8circle — COMMENTED** [review]
Thanks Yiyu!

File: src/App.js (hunk 1 above)
Comment (YiyuTimTian) [reply src/App.js]: Yes, according to the Figma design, there is not footer and header in the detail page. I can put them back if needed.

File: src/detail/detail.css (hunk 2 above)
Comment (YiyuTimTian) [reply src/detail/detail.css:1]: Yes, for sure.

File: src/detail/more/more.css (hunk 3 above)
Comment (YiyuTimTian) [reply src/detail/more/more.css]: Sure.

**Review by steve8circle — APPROVED** [review]

**Pull request merged**